### PR TITLE
Xrddev 1713 cs init e2e tests

### PR DESCRIPTION
**Pull request by teemukin65**
Imitated structure of the e2e tests from proxy-ui-api. 
Copied some custom commands from there. 
One essential 's' added to build.gradle

### CODE REVIEW

**Comment by teemukin65** [comment]
According to review cmts e2e-test spec -files renamed to ensure the test run order. Initialization tests must be run before other tests.

File: src/centralserver/admin-ui/tests/e2e/custom-commands/logMessage.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+module.exports.command = function (message) {
+  return this.perform(function (browser, done) {
+    // eslint-disable-next-line no-console
+    console.log('\033[32m✓✓✓\033[0m ' + message);
+    // eslint-disable-next-line no-console
+    console.log('\033[32m ✓✓\033[0m'); // for some reason last log message disappears
+    done();
+  });
+};
```
[the remark this answers is not in the document]
Comment (teemukin65) [reply src/centralserver/admin-ui/tests/e2e/custom-commands/logMessage.js]: done: 37fff5eea9e5cb47e3748501ef8aee671ac718cb

File: src/centralserver/admin-ui/tests/e2e/custom-commands/waitForNonEmpty.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      selectorDescForError = selector.selector;
+    }
+    this.emit(
+      'error',
+      new Error(
+        `Timeout exceeded while waiting for non-empty string value for selector [${selectorDescForError}]`,
+      ),
+    );
+  }
+};
```
[the remark this answers is not in the document]
Comment (teemukin65) [reply src/centralserver/admin-ui/tests/e2e/custom-commands/waitForNonEmpty.js:63]: done in: 37fff5eea9e5cb47e3748501ef8aee671ac718cb

File: src/centralserver/admin-ui/tests/e2e/custom-commands/waitForValue.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      selectorDescForError = selector.selector;
+    }
+    this.emit(
+      'error',
+      new Error(
+        `Timeout exceeded while waiting for value [${expectedValue}] using selector [${selectorDescForError}]`,
+      ),
+    );
+  }
+};
```
[the remark this answers is not in the document]
Comment (teemukin65) [reply src/centralserver/admin-ui/tests/e2e/custom-commands/waitForValue.js]: done in: 37fff5e

**Review by iluwatar — APPROVED** [review]

**Pull request merged**